src/renderer/views/wallet/WalletView.tsx

```diff
@@ -343,6 +340,9 @@ export const WalletView: React.FC = (): JSX.Element => {
       <Route path={walletRoutes.imports.base.template}>
         <ImportsView />
       </Route>
+      <Route path={walletRoutes.settings.template} exact>
+        <SettingsView />
+      </Route>
```

Comment:
Since `Settings` are global or app settings now, let's extract that route from `walletRoutes` into `appRoutes` using a path `/settings`. With that, route handling for `<SettingsView />` can be extracted into `<AppView />